sql/entity_mapping_type.sql

```diff
@@ -1,4 +1 @@
-INSERT IGNORE INTO entity_mapping_type
-    (name,uuid,entity1_type,entity2_type)
-VALUES
-    ('visittype_location','bf9c4e41-4b69-49b4-986d-1aafdb7dcd55','org.openmrs.VisitType','org.openmrs.Location');
+DELETE * FROM entity_mapping_type where id = '5';
```

Comment:
@prashantsable1991 There are 5 rows in entity_mapping in dev, the above INSERT added only 1 new line; so the delete query will remove records that are possibly used elsewhere. We only need to remove the records for the uuid added part of COM-37.

Reply:
@tmvumbi2 Thanks for the review. Pushed changes as requested please have a view.